infra-as-code/bicep/orchestration/subPlacementAll/README.md

```diff
@@ -91,48 +91,66 @@ This however may be done as part of another process, for example upon Subscripti
 > For the examples below we assume you have downloaded or cloned the Git repo as-is and are in the root of the repository as your selected directory in your terminal of choice.
 
 ### Azure CLI
+**NOTE: As there is some PowerShell code within the CLI, there is a requirement to execute the deployments in a cross-platform terminal which has PowerShell installed.**
 
 ```bash
 # For Azure global regions
-az deployment mg create \
-    --template-file infra-as-code/bicep/orchestration/subPlacementAll/subPlacementAll.bicep \
-    --parameters @infra-as-code/bicep/orchestration/subPlacementAll/parameters/subPlacementAll.parameters.all.json \
-    --location eastus \
-    --management-group-id alz
+
+dateYMD=$(date +%Y%m%dT%H%M%S%NZ)
+NAME="alz-SubPlacementAll-${dateYMD}"
+PARAMETERS="@infra-as-code/bicep/orchestration/subPlacementAll/parameters/subPlacementAll.parameters.all.json"
+LOCATION="eastus"
+MGID="alz"
+TEMPLATEFILE="infra-as-code/bicep/orchestration/subPlacementAll/subPlacementAll.bicep"
+
+az deployment mg create --name ${NAME:0:63} --parameters $PARAMETERS --location $LOCATION --management-group-id $MGID --template-file $TEMPLATEFILE
 ```
 
 OR
 
 ```bash
 # For Azure China regions
-az deployment mg create \
-    --template-file infra-as-code/bicep/orchestration/subPlacementAll/subPlacementAll.bicep \
-    --parameters @infra-as-code/bicep/orchestration/subPlacementAll/parameters/subPlacementAll.parameters.all.json \
-    --location chinaeast2 \
-    --management-group-id alz
+
+dateYMD=$(date +%Y%m%dT%H%M%S%NZ)
+NAME="alz-SubPlacementAll-${dateYMD}"
+PARAMETERS="@infra-as-code/bicep/orchestration/subPlacementAll/parameters/subPlacementAll.parameters.all.json"
+LOCATION="chinaeast2"
+MGID="alz"
+TEMPLATEFILE="infra-as-code/bicep/orchestration/subPlacementAll/subPlacementAll.bicep"
```

Comment:
Can we standardize the order of these so at least templatefile is then followed by parameters?